src/vblank.c

```diff
@@ -311,6 +308,35 @@ static void sgi_video_sync_scheduler_deinit(struct vblank_scheduler *base) {
 	pthread_mutex_destroy(&self->vblank_requested_mtx);
 	pthread_cond_destroy(&self->vblank_requested_cnd);
 }
+
+static void
+sgi_video_sync_scheduler_callback(EV_P attr_unused, ev_async *w, int attr_unused revents) {
+	auto sched = container_of(w, struct sgi_video_sync_vblank_scheduler, notify);
+	auto msc = atomic_load(&sched->current_msc);
+	if (sched->last_msc == msc) {
+		// NVIDIA spams us with duplicate vblank events after a suspend/resume
+		// cycle. Recreating the X connection and GLX context seems to fix this.
+		// Oh NVIDIA.
+		log_warn("Possible NVIDIA bug? Duplicate vblank event found with msc %d", msc);
```

Comment:
i'd swap the sentences here personally, i.e.:
```suggestion
		log_warn("Duplicate vblank event found with msc %d. Possible NVIDIA bug?", msc);
```

other than this works and lgtm.

Reply:
gotcha